spec/spec_helper.rb

```diff
@@ -14,6 +14,7 @@
 #
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
 require 'bundler/setup'
+require 'timecop'
```

Comment:
Why do we need this now? I can't see from the PR where and how it is necessary.

Reply:
Yes, we are using it for `report_spec.rb`, if we don't require the tests fail, in fact, they were failing before because we didn't require it for the test environment.